src/main/java/Duke.java

```diff
@@ -1,10 +1,206 @@
+            return;
+        }
+        System.out.println("____________________________________________________________");
+        System.out.println("Nice! I've marked this task as done:");
+        for (int j = 1; j < words.length; j++) {
+            int index = Integer.parseInt(words[j]) - 1;
+            tasks.get(index).markAsDone();
+            System.out.println("[" + tasks.get(index).getStatusIcon() + "] " + tasks.get(index).description);
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void listTasks(int count, ArrayList<Task> tasks) {
+        System.out.println("____________________________________________________________");
+        System.out.println("Here are the tasks in your list:");
+        for(int i = 0; i < count; i++) {
+            System.out.println((i + 1) + "." + tasks.get(i));
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void greet() {
+        System.out.println("____________________________________________________________");
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void deleteTasks(ArrayList<Task> tasks, String userInput, int count) {
+        String[] words = userInput.split(" ");
+        switch (words.length) {
+            case 1:
+                System.out.println("____________________________________________________________");
+                System.out.println(" ☹ OOPS!!! You are not deleting any task.");
+                System.out.println("____________________________________________________________");
+                break;
+            case 2:
+                int index = Integer.parseInt(words[1]) - 1;
+                System.out.println(tasks.get(index));
+                tasks.remove(index);
+                System.out.println("____________________________________________________________");
+                System.out.println("Noted. I've removed this task:");
+                System.out.println(("Now you have " + count + " tasks in the list").replace("1 tasks", "1 task"));
+                System.out.println("____________________________________________________________");
+                break;
+            default:
+                System.out.println("____________________________________________________________");
+                System.out.println(" ☹ OOPS!!! Please enter just one number after 'delete' command.");
+                System.out.println("____________________________________________________________");
+                break;
+        }
+    }
+
+    private static void writeToFile(String filePath, ArrayList<Task> textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
```

Comment:
Maybe you should consider using "fileWriter" as the variable name instead of "fw".